File: lab-03/src/main/java/by/lenson423/paint/RadioButtonWithBiFunction.java
package by.lenson423.paint;

import javafx.scene.control.RadioButton;

import java.util.function.BiFunction;

public class RadioButtonWithBiFunction<T, U> extends RadioButton {
    private BiFunction<T, U, Void> function;

    void setFunction(BiFunction<T, U, Void> function){
        this.function = function;
    }

    BiFunction<T, U, Void> getFunction(){
        return function;
    }
}

File: lab-03/src/main/java/by/lenson423/paint/RadioButtonWithBiFunction.kt
package by.lenson423.paint

import javafx.scene.control.RadioButton
import java.util.function.BiFunction

class RadioButtonWithBiFunction<out T, out U> : RadioButton() {
    var function: BiFunction<@UnsafeVariance T, @UnsafeVariance U, Void?>? = null
}
